iot_verifier/src/settings.rs

```diff
@@ -40,6 +40,41 @@ pub struct Settings {
     /// Tolerance applied to beacon intervals within which beacons will be accepted (in seconds)
     #[serde(default = "default_beacon_interval_tolerance")]
     pub beacon_interval_tolerance: i64,
+    /// window width for the entropy and poc report loader ( in seconds )
```

Comment:
Don't forget to update the settings template toml file with these, including their comments and requirements like for `default_poc_loader_window_max_lookback_age`